x-pack/plugins/security_solution/cypress/screens/document_expandable_flyout.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  ANALYSER_GRAPH_TEST_ID,
+  SESSION_VIEW_TEST_ID,
+} from '../../public/flyout/left/components/test_ids';
+import {
+  HISTORY_TAB_CONTENT_TEST_ID,
+  INSIGHTS_TAB_CONTENT_TEST_ID,
+  INVESTIGATIONS_TAB_CONTENT_TEST_ID,
+  VISUALIZE_TAB_BUTTON_GROUP_TEST_ID,
+  VISUALIZE_TAB_GRAPH_ANALYSER_BUTTON_TEST_ID,
+  VISUALIZE_TAB_SESSION_VIEW_BUTTON_TEST_ID,
+} from '../../public/flyout/left/tabs/test_ids';
+import {
+  HISTORY_TAB_TEST_ID,
+  INSIGHTS_TAB_TEST_ID,
+  INVESTIGATIONS_TAB_TEST_ID,
+  VISUALIZE_TAB_TEST_ID,
+} from '../../public/flyout/left/test_ids';
+import {
+  JSON_TAB_TEST_ID,
+  OVERVIEW_TAB_TEST_ID,
+  TABLE_TAB_TEST_ID,
+} from '../../public/flyout/right/test_ids';
+import {
+  JSON_TAB_CONTENT_TEST_ID,
+  OVERVIEW_TAB_CONTENT_TEST_ID,
+  TABLE_TAB_CONTENT_TEST_ID,
+} from '../../public/flyout/right/tabs/test_ids';
+import {
+  COLLAPSE_DETAILS_BUTTON_TEST_ID,
+  EXPAND_DETAILS_BUTTON_TEST_ID,
+  FLYOUT_HEADER_TITLE_TEST_ID,
+} from '../../public/flyout/right/components/test_ids';
+
+export const DOCUMENT_DETAILS_FLYOUT_HEADER_TITLE = `[data-test-subj="${FLYOUT_HEADER_TITLE_TEST_ID}"]`;
```

Comment:
As a thought for our cypress tests, rather than repeating all of the `[data-test-subj="blah"]`, why not just make a helper `const getTestSelector = (dataTestSubjectValue: string) => [data-test-subj="${dataTestSubjectValue}"]`? Might be useful for all of our selectors tbh

Reply:
good idea, I extracted this in a function that I put under `/helpers/common.ts`, not sure if there is a better place...